docs/configuration/overview.mdx

```diff
@@ -183,7 +183,7 @@ dependencyOverridePaths:
 
 Whether to run `pub get` in parallel during bootstrapping.
 
-The default is `true`.
+The default is `auto`, which fallbacks to `false` if the machine is a CI instance.
```

Comment:
If this change will make into main, I would suggest adding explanation to this line. Why is this the default, what is the issue it resolves, and what could someone who changes this to true expect.

Reply:
Can you look into that @tomassasovsky?